src/gui/TelaEmprestimoDetalheController.java

```diff
@@ -0,0 +1,88 @@
+package gui;
+
+import gerenciamento.SessionManager;
+import gui.models.MovimentacaoModelo;
+import javafx.fxml.FXML;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TextField;
+import javafx.scene.control.cell.PropertyValueFactory;
+import negocio.Fachada;
+import negocio.beans.Bens;
+import negocio.beans.Cliente;
+import negocio.beans.Emprestimo;
+import negocio.beans.Movimentacao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TelaEmprestimoDetalheController {
+    @FXML TextField txtNomeCliente;
+    @FXML DatePicker dtData;
+    @FXML TextField txtValor;
+    @FXML TextField txtParcelas;
+    @FXML DatePicker dtPrazo;
+
+    @FXML TableView<MovimentacaoModelo> tblvExtrato;
+    @FXML TableColumn<MovimentacaoModelo, Double> colParcelas;
+    @FXML TableColumn<MovimentacaoModelo, String> colValorParcela;
+    @FXML TableColumn<MovimentacaoModelo, String> colDataVencimento;
+    @FXML TableColumn<MovimentacaoModelo, String> colDataPagamento;
+    @FXML TableColumn<MovimentacaoModelo, String> colTipoPagamento;
+    
+    @FXML ListView<String> lstvGarantias;
+
+    private List<Bens> listGarantiasTemp = new ArrayList<>();
+    
+    @FXML
+    private void initialize() {
+        this.initializeTableViews();
+        List<Movimentacao> movimentacaoList = new ArrayList<>(Fachada.getInstance().listarMoveCliente(SessionManager.getInstance().getPessoaSessao().getUid()).values());
+        this.atualizarTableViewExtrato(movimentacaoList);
+        
+        this.atualizarListViewGarantias();
```

Comment:
Você poderia ter colocado essa instrução dentro de `this.initializeDetalhesDoEmprestimo()`

Reply:
Boa ideia!